From 25614fbd2c0b1e097b6ab27115f25031dc60b986 Mon Sep 17 00:00:00 2001
From: Alfred Wingate <parona@protonmail.com>
Date: Tue, 10 Oct 2023 06:04:13 +0300
Subject: [PATCH] bluetooth.ver: remove undefined symbols

* error and debug only exists macros and not as symbols.

Signed-off-by: Alfred Wingate <parona@protonmail.com>
---
 src/bluetooth.ver | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/bluetooth.ver b/src/bluetooth.ver
index a96fda2a15..18e121c21b 100644
--- a/src/bluetooth.ver
+++ b/src/bluetooth.ver
@@ -3,8 +3,6 @@
 		btd_*;
 		g_dbus_*;
 		info;
-		error;
-		debug;
 		baswap;
 		ba2str;
 		/* Don't break LLVM sanitizers */
